platforms/ide/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/BuildInvocationsBuilder.java

```diff
@@ -71,11 +73,11 @@ public DefaultBuildInvocations buildAll(String modelName, Project project) {
         for (String selectorName : selectorsByName.keySet()) {
             LaunchableGradleTaskSelector selector = selectorsByName.get(selectorName);
             selectors.add(selector.
-                    setName(selectorName).
-                    setTaskName(selectorName).
-                    setProjectIdentifier(projectIdentifier).
-                    setDisplayName(selectorName + " in " + project + " and subprojects.").
-                    setPublic(visibleTasks.contains(selectorName)));
+                setName(selectorName).
```

Comment:
🎉  Kudos for reformatting this. I would go one step further and also replace "trailing dots" with "leading dots" like
```
.setName(selectorName)
```

I think we consistently use the latter across the codebase except for some older places